Sources/PayPal/Buttons/PayPalCreditButton.swift

```diff
@@ -0,0 +1,24 @@
+import UIKit
+
+// Configuration for PayPal Credit button
```

Comment:
This looks like a comment and not a docstring?

Reply:
Also seeing this pattern in all of the new files.

Reply:
Updated!